libs/payments/ui/src/lib/server/subscription-title.ftl

```diff
@@ -0,0 +1,8 @@
+## Component - SubscriptionTitle
+
+next-subscription-create-title = Set up your subscription
+next-subscription-success-title = Subscription confirmation
+next-subscription-processing-title = Confirming subscription…
+next-subscription-error-title = Error confirming subscription…
+
+next-sub-guarantee = 30-day money-back guarantee
```

Comment:
Should this be `-sub-` or `-subscription-` for consistency?
Or is it "-sub-" for "subtitle"?

Reply:
To ensure we aren't duplicating existing strings, we are adding the prefix `next-` to ones we are migrating to the new library.

Reply:
Sorry, I think I was more confused as to whether "sub" in the middle meant ("subscription" or "subtitle").

Reply:
I am guessing when `sub-guarantee` was created that `-sub-` referred to "subtitle". I think it would've been written out as `-subscription-` for consistency. While I'd prefer to update it, it wouldn't be ideal for this existing string to go through the translation process again.